api/Screen.json

```diff
@@ -50,59 +51,1084 @@
+            },
+            "edge": {
+              "version_added": true,
+              "notes": "Only <code>height</code> and <code>width</code> reflect the current screen; <code>availHeight</code> and <code>availWidth</code> always reflect the main screen; <code>availLeft</code> and <code>availTop</code> are not implemented."
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "addEventListener": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/EventTarget/addEventListener",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true,
+              "notes": "Only <code>height</code> and <code>width</code> reflect the current screen; <code>availHeight</code> and <code>availWidth</code> always reflect the main screen; <code>availLeft</code> and <code>availTop</code> are not implemented."
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "removeEventListener": {
```

Comment:
This PR includes a bunch of inherited methods from EventTarget that shouldn't be included